packages/kbn-plugin-generator/sao_template/template/public/index.ts

```diff
@@ -0,0 +1,18 @@
+<%_ if (hasScss) { -%>
+import './index.scss';
+<%_ } -%>
+
+import { <%= upperCamelCaseName %>Plugin } from './plugin';
+
+/**
+ * This exports static code and TypeScript types, 
+ * as well as, Kibana Platform `plugin()` initializer.
+ **/ 
+export function plugin() {
+  return new <%= upperCamelCaseName %>Plugin();
+}
```

Comment:
This is a file description comment right? I think `/** **/ ` should be changed to `/* */` as atm it's considered as jsdoc for the `plugin` export

Reply:
It's actually just a comment for the user generating the plugin.
I'm switching it to // comments to avoid confusion.